data/mods/CrazyCataclysm/crazy_effects.json

```
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "effect_type",
+    "id": "mutagen_troglobite",
+    "name": [ "Gamer Mutation", "Gamer Transformation", "Gamer Metamorphosis" ],
```

More out there names perhaps?
```suggestion
    "name": [ "Gamer Juice", "Gamer Fuel", "Gamer Thirst" ],
```